### PR TITLE
Rails 5.1 support

### PR DESCRIPTION
**Pull request by lleirborras**
Nowadays people has this obsession on updating to the latest versions of software leaving poor nightly workers to adapt legacy over engineered software to it's new interfaces with pain and suffering while watching some b-series movie in a CRT monitor glorifying old days in software development.
image

### CODE REVIEW

**Comment by vjt** [comment]
LGTM - if you can add a brief description of the pull that would be fruitful for future reference.

Thanks

**Comment by lleirborras** [comment]
@vjt all green, please complete review

File: lib/eaco/adapters/active_record/compatibility/sanitized.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        ##
+        # Aliases `sanitize` as `connection.quote` for ActiveRecord >= 5.1.
+        #
+        module Sanitized
+
+          ##
+          # Just returns +ActiveRecord::Base.connection.quote+.
+          #
+          def sanitize(d)
+            ::ActiveRecord::Base.connection.quote(d)
```
Comment (vjt) [inline lib/eaco/adapters/active_record/compatibility/sanitized.rb]: can we avoid the explicit reference to AR::Base?

**Comment by pzac** [comment]
:+1:

File: lib/eaco/adapters/active_record/compatibility/v51.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+        ##
+        # Rails 5.1 support module.
+        #
+        # JSONB works correctly, but we need +.scoped+ so we revive it through
+        # the {Scoped} support module.
+        #
+        # Sanitize has dissapeared in favour of quote.
+        #
+        # @see Scoped
```
Comment (vjt) [inline lib/eaco/adapters/active_record/compatibility/v51.rb]: Please add a `@see Sanitized`

File: lib/eaco/adapters/active_record/compatibility/sanitized.rb (hunk 1 above)
Comment (lleirborras) [inline lib/eaco/adapters/active_record/compatibility/sanitized.rb]: Fixed https://github.com/ifad/eaco/pull/12/commits/a001e7b89ac9714a36d638657f46365fc6f3f737

File: lib/eaco/adapters/active_record/compatibility/v51.rb (hunk 2 above)
Comment (lleirborras) [inline lib/eaco/adapters/active_record/compatibility/v51.rb]: Fixed https://github.com/ifad/eaco/pull/12/commits/ef224ed86ff228795d98e677cfd86cff21d00082

**Comment by lleirborras** [comment]
@vjt All your comments have been taken in consideration. Please review

**Comment by lleirborras** [comment]
@vjt all green, feel free to merge

**Pull request merged**